evalai/worker_config.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+
+
+def task_wrapper(fn, *args, **kwargs):
+    def inner(fn):
+        if settings.DEBUG:
+            import dramatiq  # noqa
+            from evalai.dramatiq_conf import broker  # noqa
```

Comment:
Can you explain the reason of why is this not on top and here?

Reply:
@vkartik97 The dramatiq import isn't required in the other case in prod env. 
Also, we don't have dramatiq in `requirements/prod.txt` so having it at the top would cause an error in prod and test envs.

Reply:
Can this moved before `def task_wrapper` ?

Reply:
Oh right! That will make the code much cleaner! 😄 
Thanks for the suggestion that had slipped my mind.

Reply:
Done! :+1: